twisted_tools/scripts/forcebuild.py

```diff
@@ -1,9 +1,9 @@
 """
-Force the Twisted buildmaster to run a builds on all supported builders for
-a particular branch.
+Command line adapter for L{buildot.forceBuild}.
```

Comment:
This looks like a *less* useful docstring to me, as it goes from saying what it does to saying which piece of code it's invoking.